### PR TITLE
Update awards for all expiry email

### PR DESCRIPTION
**Pull request by davidrapson**
Revises the copy to avoid saying 'application' over and over again and to simplify some of the language used.

Have sent the latest copy for translation

### CODE REVIEW

**Comment by davidrapson** [comment]
[Check A4A reminder email to see how many times it says 'application'](https://trello.com/c/7M8VLJCw/1060-check-a4a-reminder-email-to-see-how-many-times-it-says-application)

File: controllers/apply/expiries/views/awards-for-all-expiry-email.njk [hunk 1]
```diff
@@ -1,45 +1,45 @@
...
     </p>
-
     <p>
-        <strong>You have until {{ expiryDate.en }} to finish your application.</strong><br/>
-        If you don’t complete your application before that time, we’ll delete it - so that we can keep your data safe.
+        <strong>
+            You have until {{ expiryDate.en }}
+            to finish your application
+        </strong><br/>
+        If you don’t complete it at that time,
```
Comment (mattandrews) [inline controllers/apply/expiries/views/awards-for-all-expiry-email.njk]: Should this say "before that time"?

Comment (davidrapson) [reply controllers/apply/expiries/views/awards-for-all-expiry-email.njk]: Good shout, this is what we had before, I've updated this to keep that phrasing.

**Comment by davidrapson** [comment]
@mattandrews Mind giving this another look. Added latest welsh translation and spotted that we could do with a snapshot test for the bilingual emails too.

**Comment by davidrapson** [comment]
This is what the final updated version looks like for reference:

image

**Pull request merged**